azure-webapp-maven-plugin/README.md

```diff
@@ -155,9 +157,7 @@ Property | Required | Description | Version
     Users don't need to care about the deployment type in v2 configuration.
     Just configure the resources to deploy to the Web App.
     
-    It will use the zip deploy for fast and easy deploy.
-    But if the artifact(s) are war package(s), war deploy will be used.
-    Mix deploying war packages and other kinds of artifacts is not suggested and will cause errors.
+    By default, it will use the zip deploy which is fast and easy to complete the deployment. But if the artifact(s) are war package(s), war deploy will be used.Don't suggest mixing war packages and other kinds of artifacts.
```

Comment:
Add a link to zip deploy and war deploy.
- Zip Deploy: https://github.com/projectkudu/kudu/wiki/Deploying-from-a-zip-file
- War Deploy: https://github.com/projectkudu/kudu/wiki/Deploying-WAR-files-using-wardeploy